test_run.lua

```diff
@@ -79,8 +79,13 @@ local function wait_lsn(self, waiter, master)
     end
 end
 
-local function get_vclock(self, node)
-    return self:get_param(node, 'vclock')[1]
+local function get_vclock(self, node, opts)
+    local opts = opts or {}
+    local vclock = self:get_param(node, 'vclock')[1]
+    if opts.ignore_zero then
```

Comment:
Wouldn't `opts.ignore_local` be a better name?

Reply:
I've used same name from core https://github.com/tarantool/tarantool/blob/a24714c55305280838576fa6dec9a9781763e893/src/lib/vclock/vclock.h#L285 I can rename it, if you insist.

Reply:
Ok.